afkak/client.py

```diff
@@ -662,8 +669,13 @@ def _cancel_timeout(_, dc):
             # Set a delayedCall to fire if we don't get a reply in time
             dc = self._get_clock().callLater(
                 self.timeout, _timeout_request, broker, requestId)
-            # Setup a callback on the request deferred to cancel timeout
+            # Set a delayedCall to complain if the reactor has been blocked
+            rc = self._get_clock().callLater(
+                (self.timeout * 0.9), _alert_blocked_reactor, self.timeout,
+                time.time())
```

Comment:
Switching to `reactor.seconds()` might have made writing the test easier though. :-)  I'll have to go back and look at that test.